### PR TITLE
Add v1.15.0 images

### PR DESCRIPTION
**Pull request by ashie**
And update alpine image, jemalloc, msys, and gems.
We should also merge #330.

In addition, I'm considering to update Ruby to 3.1.
Do you have any concern about it? @cosmo0920

### CODE REVIEW

**Comment by ashie** [comment]
I'll address remaining issues later...

File: v1.15/alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        ruby ruby-irb ruby-etc ruby-webrick \
+        tini \
+ && apk add --no-cache --virtual .build-deps \
+        build-base linux-headers \
+        ruby-dev gnupg \
+ && echo 'gem: --no-document' >> /etc/gemrc \
+ && gem install oj -v 3.10.18 \
+ && gem install json -v 2.4.1 \
+ && gem install async-http -v 0.54.0 \
+ && gem install ext_monitor -v 0.1.2 \
```
Comment (cosmo0920) [inline v1.15/alpine/Dockerfile:23]: We must remove `ext_monitor` installation step if we move to use Ruby 3.0 or later.
ref: https://rubygems.org/gems/ext_monitor

File: v1.15/arm64/debian/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ && buildDeps=" \
+      make gcc g++ libc-dev \
+      wget bzip2 gnupg dirmngr \
+    " \
+ && apt-get install -y --no-install-recommends $buildDeps \
+ && echo 'gem: --no-document' >> /etc/gemrc \
+ && gem install oj -v 3.10.18 \
+ && gem install json -v 2.4.1 \
+ && gem install async-http -v 0.54.0 \
+ && gem install ext_monitor -v 0.1.2 \
```
Comment (cosmo0920) [inline v1.15/arm64/debian/Dockerfile:35]: Ditto.

File: v1.15/debian/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ && buildDeps=" \
+      make gcc g++ libc-dev \
+      wget bzip2 gnupg dirmngr \
+    " \
+ && apt-get install -y --no-install-recommends $buildDeps \
+ && echo 'gem: --no-document' >> /etc/gemrc \
+ && gem install oj -v 3.10.18 \
+ && gem install json -v 2.4.1 \
+ && gem install async-http -v 0.54.0 \
+ && gem install ext_monitor -v 0.1.2 \
```
Comment (cosmo0920) [inline v1.15/debian/Dockerfile:24]: Ditto.

File: v1.15/armhf/debian/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ && buildDeps=" \
+      make gcc g++ libc-dev \
+      wget bzip2 gnupg dirmngr \
+    " \
+ && apt-get install -y --no-install-recommends $buildDeps \
+ && echo 'gem: --no-document' >> /etc/gemrc \
+ && gem install oj -v 3.10.18 \
+ && gem install json -v 2.4.1 \
+ && gem install async-http -v 0.54.0 \
+ && gem install ext_monitor -v 0.1.2 \
```
Comment (cosmo0920) [inline v1.15/armhf/debian/Dockerfile:35]: Ditto.

**Review by cosmo0920 — CHANGES_REQUESTED** [review]

**Comment by cosmo0920** [comment]
Our CI is blocked in ext_monitor gem installation: https://github.com/fluent/fluentd-docker-image/runs/7227402721?check_suite_focus=true#step:3:125
This gem needn't install on Ruby 3.0 or later.

File: v1.15/alpine/Dockerfile (hunk 1 above)
Comment (ashie) [reply v1.15/alpine/Dockerfile]: I updated Ruby to 3.1 and removed ext_monitor. Thanks

**Review by cosmo0920 — APPROVED** [review]

**Comment by cosmo0920** [comment]
@ashie Can we marge this PR? I'd like to add my commits on #330 for Fluentd v1.15.0 image release.

**Pull request merged**